datalad/support/tests/test_annexrepo.py

```diff
@@ -1134,7 +1134,8 @@ def test_annex_copy_to(origin, clone):
     repo.get("test-annex.dat")
     # now it has:
     eq_(repo.copy_to("test-annex.dat", "target"), ["test-annex.dat"])
-    eq_(repo.copy_to(["INFO.txt", "test-annex.dat"], "target"), ["test-annex.dat"])
+    # and will not be copied again since it was already copied
+    eq_(repo.copy_to(["INFO.txt", "test-annex.dat"], "target"), [])
```

Comment:
thanks @bpoldrack -- good arguments... will address them whenever get to this 
as for --json and non-existing files -- see http://git-annex.branchable.com/bugs/copy_does_not_reflect_some_failed_copies_in_--json_output/  which is relevant here... I guess while processing stderr coming from `--json` we need also to catch those :-/